openshift/templates/c_haines.cronjob.yaml

```diff
@@ -0,0 +1,119 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: ${JOB_NAME}-cronjob-template
+  annotations:
+    description: "Scheduled task to run c-haines."
+    tags: "cronjob,c-haines"
+labels:
+  app.kubernetes.io/part-of: "${NAME}"
+  app: ${NAME}
+parameters:
+  - name: NAME
+    description: Module name
+    # value: wps-c-haines
+    value: wps
+  - name: IMAGE_REGISTRY
+    description: Image registry for imagestream.
+    value: image-registry.openshift-image-registry.svc:5000
+    required: true
+  - name: IMAGE_NAME
+    required: true
+    # value: ${NAME}-${SUFFIX}
+  - name: IMAGE_TAG
+    required: true
+    # value: ${SUFFIX}
```

Comment:
did you intend to leave the default values for NAME, IMAGE_TAG, and IMAGE_NAME commented out?

Reply:
Removed it.